war/src/main/java/io/lumeer/engine/rest/DocumentService.java

```diff
@@ -130,12 +128,16 @@ public void init() {
    @Consumes(MediaType.APPLICATION_JSON)
    public String createDocument(final @PathParam("collectionName") String collectionName, final DataDocument document) throws DbException, InvalidConstraintException, InvalidValueException {
       if (collectionName == null || document == null) {
-         throw new BadRequestException();
+         throw new IllegalArgumentException();
```

Comment:
I changed it because all other methods in the class throw `IllegalArgumentException`.